packages/core/.eslintrc.cjs

```diff
@@ -22,6 +22,8 @@ module.exports = {
   rules: {
     // TODO(mc, 2023-12-15): remove overrides, fix issues
     '@typescript-eslint/no-non-null-assertion': 'warn',
+    // TODO(mc, 2024-01-03): move to base config?
+    'multiline-comment-style': 'off',
```

Comment:
What do we think about this? This lint adds a lot of friction without any real payoff, in my opinion. Sometimes I just need to write a two-line TODO comment, and this blows it up to four lines. Plus, it'd be nice to reserve `/* */` for JSDoc

Reply:
Agreed all around with this

Reply:
https://github.com/viamrobotics/js-config/pull/34